specification/datalake-store/data-plane/Microsoft.DataLakeStore/preview/2015-10-01-preview/filesystem.json

```diff
@@ -277,8 +277,10 @@
           }
         }
       }
-    },
-    "/webhdfs/v1/{destinationPath}": {
+    }
+  },
+  "x-ms-paths": {
+    "/webhdfs/v1/{destinationPath}?op=CONCAT": {
```

Comment:
Is there a process to deprecate this Concat. We should only have MSConcat. Concat has some problem for which customer faces issues.

Reply:
Yes we can add `deprecated: true` and update the description string.  You want me to do that?

Reply:
Yes I think we should do that. Concat takes in input as queryparameters which causes another IIS filtering 404 error which is notoriously difficult to catch.

Reply:
Done, see latest commit.